src/transformers/training_args_tf.py

```diff
@@ -110,12 +110,23 @@ class TFTrainingArguments(TrainingArguments):
         metadata={"help": "Name of TPU"},
     )
 
+    xla: bool = field(default=False, metadata={"help": "Whether to activate the XLA compilation or not"})
```

Comment:
Can you document it in the docstring too?

Reply:
Should be ok.